lstchain/mc/plot_utils.py

```diff
@@ -125,8 +130,8 @@ def sens_minimization_plot(eb, gb, tb, e, sens):
     Returns
     --------
     """
-    if (eb == 12):
-        figarr, axarr = plt.subplots(4,3, sharex=True, sharey=True, figsize=(13.2,18))
+    # if (eb == 12):
+    #     figarr, axarr = plt.subplots(4,3, sharex=True, sharey=True, figsize=(13.2,18))
 
     # To be changed!!!
```

Comment:
could you move the comment up, before the commented lines? So I do not forget what is it that needs to be changed?

Reply:
done